WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModel.kt

```diff
@@ -348,17 +336,18 @@ class SiteCreationDomainsViewModel @Inject constructor(
         )
     }
 
+    @Suppress("ForbiddenComment")
```

Comment:
I think the `Suppress` annotation is not needed in this function

Reply:
Indeed, Thomas noticed it as well 👍 
Thanks, I removed it in commit c96825a7